bodhi/server/templates/update.html

```diff
@@ -680,7 +680,7 @@ <h4>Reboot Required</h4>
           % endif
 
           % if update.content_type:
-          % if update.content_type.value == 'rpm' and ('testing' in update.status or 'stable' in update.status):
+          % if update.content_type.value == 'rpm' and ('testing' in update.status or 'stable' in update.status) and not (update.release.package_manager.value == 'unspecified' or update.release.testing_repository is None):
```

Comment:
Maybe we can avoid the double test on `content_type`, since we now have the check on `package-manager` and `testing-repository`.

Reply:
Yeah I think we could avoid it, especially since we've ensured that it's dnf, yum, or unspecified. We can leave it up to admins to only set this where it makes sense.

Also, we should compare Enums directly rather than comparing the string values.